Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
```

Comment:
Instead of this concatenation, a `$detectedPathsParts` array could be used, and then imploded with `.`